### ISSUE TITLE
EmptyFunctionBlock should not flag functions with open modifier

### ISSUE THREAD

**Issue by egor-n** [arturbosch/detekt#666]
The title duplicates this test:
https://github.com/arturbosch/detekt/blob/994c732c5f45ebdbb50a917b272f87a9c0f5916f/detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/empty/EmptyFunctionBlockSpec.kt#L29-L32

The name is correct, the implementation is not but it passes. The test should read:
```kotlin
assertThat(findings).isEmpty()
```
